server/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "nodemon server.js"
```

Comment:
Good add. I think there's a way to add it as a "dev-dependency", too.

Reply:
Oh I probably forgot to add it in as a --save 😅